### PR TITLE
replace path separators in log stream name with dashes

### PR DESCRIPTION
**Pull request by toebbel**
I have log streams in AWS Cloudwatch that contain slashes.

### CODE REVIEW

**Comment by hokkai7go** [comment]
@ryotarai 
I have same problem.
doing the same way even in my environment, so I want you to merge this pull request.
Is there anything that blocks the merge?

File: lib/fluent/plugin/in_cloudwatch_logs.rb [hunk 1]
```diff
@@ -87,28 +90,27 @@ def run
...
+    def emit(stream, event)
       if @parser
         record = @parser.parse(event.message)
-        router.emit(@tag, record[0], record[1])
+        router.emit("#{@tag}.#{stream}", event.timestamp / 1000, record[1])
       else
-        time = (event.timestamp / 1000).floor
         record = JSON.parse(event.message)
-        router.emit(@tag, time, record)
+        router.emit("#{@tag}.#{stream}", event.timestamp / 1000, record)
```
Comment (hokkai7go) [inline lib/fluent/plugin/in_cloudwatch_logs.rb]: Why add `#{stream}` ?
I'd like tag to be unchanged

File: lib/fluent/plugin/cloudwatch/logs/version.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ module Fluent
   module Plugin
     module Cloudwatch
       module Logs
-        VERSION = "0.3.3"
+        VERSION = "0.3.4"
```
Comment (ryotarai) [inline lib/fluent/plugin/cloudwatch/logs/version.rb]: Please leave this line

File: lib/fluent/plugin/in_cloudwatch_logs.rb (hunk 1 above)
Comment (ryotarai) [reply lib/fluent/plugin/in_cloudwatch_logs.rb]: +1

**Comment by toebbel** [comment]
I'm very sorry that I did not react to the PR review changes. Thanks @hokkai7go for picking it up where I left off.

closing because duplicate of #80

**Pull request closed without merging**